plugins/inputs/webhooks/webhooks.go

```diff
@@ -0,0 +1,81 @@
+package webhooks
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	_ "github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_all"
+	"github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_models"
+)
+
+func init() {
+	inputs.Add("webhooks", func() telegraf.Input { return NewWebhooks() })
+}
+
+type Webhooks struct {
+	ServiceAddress string
+
+	Webhook []WebhookConfig
+}
+
+type WebhookConfig struct {
```

Comment:
This could be problematic as future webhooks could likely have very different necessary configuration parameters.

I would prefer if we could directly define different types of webhooks here. This would introduce the limitation that only one type of webhook would be allowed per-port, but I think this is OK.

ie, config would look like this:

```toml
[[inputs.webhook]]
  service_address = ":1619"
  [[inputs.webhooks.github]]
    path = "/github"
  [[inputs.webhooks.rollbar]]
    path = "/rollbar"
```